common/rpc/rpc.go

```diff
@@ -251,6 +258,14 @@ func (d *RPCFactory) dial(hostName string, tlsClientConfig *tls.Config, dialOpti
 }
 
 func (d *RPCFactory) getClientKeepAliveConfig(serviceName primitives.ServiceName) grpc.DialOption {
+	if !d.dynamicConfig.EnableInterNodeClientKeepAlive() {
+		// default keepalive settings for clients
+		return grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			Time:                time.Duration(math.MaxInt64),
+			Timeout:             20 * time.Second,
+			PermitWithoutStream: false,
+		})
+	}
 	serviceConfig := d.config.Services[string(serviceName)]
 	return grpc.WithKeepaliveParams(serviceConfig.RPC.ClientConnectionConfig.GetKeepAliveClientParameters())
```

Comment:
maybe write this reversed, like:
```suggestion
	// default keepalive settings for clients
	params := keepalive.ClientParameters{
		Time:                time.Duration(math.MaxInt64),
		Timeout:             20 * time.Second,
		PermitWithoutStream: false,
	}
	if d.dynamicConfig.EnableInterNodeClientKeepAlive() {
		serviceConfig := d.config.Services[string(serviceName)]
		params = serviceConfig.RPC.ClientConnectionConfig.GetKeepAliveClientParameters()
	}
	return grpc.WithKeepaliveParams(params)
```